contracts/citizens/DeveloperSkillLevels.sol

```diff
@@ -98,5 +104,19 @@ contract DeveloperSkillLevels {
         skillLevelAverages[developer] = newSkillLevelAverage;
         skillLevelRatings[developer][msg.sender] = rating;
         emit Rated(developer, rating, msg.sender);
+
+        eas.attest(
+            AttestationRequest({
+                schema: easSchemaUID,
+                data: AttestationRequestData({
+                    recipient: developer,
+                    expirationTime: NO_EXPIRATION_TIME,
+                    revocable: false,
+                    refUID: EMPTY_UID,
+                    data: abi.encode(0), // TO DO
+                    value: 0
+                })
+            })
+        );
```

Comment:
I am intrigued by this. Why in GWEI?

`uint256 ratingInGwei = rating * 1 ether;`

Reply:
@johnmark13 This is needed to calculate the number in decimals (3.33333333 instead of just 3 or 4).  We could just store the integers, but it's probably more interesting for citizens if they can see a more granular average evolving over time.